release/e2e.py

```diff
@@ -771,11 +800,9 @@ def create_and_wait_for_session(
 def run_session_command(sdk: AnyscaleSDK,
                         session_id: str,
                         cmd_to_run: str,
-                        stop_event: multiprocessing.Event,
                         result_queue: multiprocessing.Queue,
                         env_vars: Dict[str, str],
-                        state_str: str = "CMD_RUN",
-                        kick_off_only: bool = False) -> Tuple[str, int]:
+                        state_str: str = "CMD_RUN") -> Tuple[str, int]:
```

Comment:
Is `Tuple[str, int]` still the right return type? Maybe `Tuple[str, Any]`?

Reply:
Let me make a follow up PR